### DISCUSSION TITLE
Issue with Email Address Displayed on Vercel Platform

### DISCUSSION THREAD

**Discussion by Iamsaadanwar** [vercel/community#6079] [Help] [answered]
### Summary

Hi

I recently migrated all my DNS settings from my Zoho account to Vercel, following the instructions provided in your Learn section. While I am able to send and receive emails successfully, there is an issue with the display of the sender email address.

Instead of showing the email address I have set up on Zoho, which is [YourEmailAddress@YourDomain.com], the recipient sees an email address in the format of "1709951419489_20094175758@zohomail.eu".

This discrepancy is concerning, as I rely on my domain email for communication with clients, and the current display does not convey a professional image.

I have followed the instructions meticulously, but the issue persists. Could you please assist me in identifying the root cause of this problem and provide guidance on how to resolve it as soon as possible? Ensuring that my domain email works correctly is crucial for my business operations.

Thank you for your attention to this matter. I appreciate your prompt assistance in resolving this issue.

Best regards,

### Steps to Reproduce

I hope this email finds you well. I am writing to address an issue I am experiencing with my email setup on the Vercel platform.

I recently migrated all my DNS settings from my Zoho account to Vercel, following the instructions provided in your Learn section. While I am able to send and receive emails successfully, there is an issue with the display of the sender email address.

Instead of showing the email address I have set up on Zoho, which is [YourEmailAddress@YourDomain.com], the recipient sees an email address in the format of "1709951419489_20094175758@zohomail.eu".

This discrepancy is concerning, as I rely on my domain email for communication with clients, and the current display does not convey a professional image.

I have followed the instructions meticulously, but the issue persists. Could you please assist me in identifying the root cause of this problem and provide guidance on how to resolve it as soon as possible? Ensuring that my domain email works correctly is crucial for my business operations.

Thank you for your attention to this matter. I appreciate your prompt assistance in resolving this issue.

**Comment by amyegan** [comment, marked as answer]
Hey @Iamsaadanwar. This appears to be caused by how Zoho works rather than a DNS issue. Please contact Zoho for help with email configuration. 🙏
